.gitignore

```diff
@@ -5,8 +5,5 @@
 **/.vscode/**
 **/.idea/**
 
-# package-lock of packages doesn't make sense with Lerna
-**/package-lock.json
```

Comment:
nit: why removed?

Reply:
Actually this one is already committed to master, but appears also here so I'll just rebase.
Anyway the reason for it is that I had to commit the package-lock.json files in order to build salto with this dependency:

> lerna ERR! npm ci --no-package-lock exited 1 in '@oracle/suitecloud-cli'
lerna ERR! npm ci --no-package-lock stderr:
npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.